parallel_algorithms/P3179/P3179.md

```diff
@@ -3855,16 +3865,16 @@ every iterator `i` in the range `[first1, last1)`. Otherwise, returns `false`.
 - [3.1]{.pnum} the types of `first1`, `last1`, `first2`, and `last2` meet the *Cpp17RandomAccessIterator* requirements
   ([random.access.iterators]{- .sref}) and `last1 - first1 != last2 - first2` for the overloads in namespace `std`;
 - [3.2]{.pnum} the types of `first1`, `last1`, `first2`, and `last2` pairwise model `sized_sentinel_for`
-  ([iterator.concept.sizedsentinel]{- .sref}) and `last1 - first1 != last2 - first2` for the first overload in namespace `ranges`,
+  ([iterator.concept.sizedsentinel]{- .sref}) and `last1 - first1 != last2 - first2` for the first overload in namespace `ranges`, [or]{.add}
 - [3.3]{.pnum} `R1` and `R2` each model `sized_range` and `ranges::distance(r1) != ranges::distance(r2)` for the second
   overload in namespace `ranges`,
 
 then no applications of the corresponding predicate and each projection; otherwise,
 
-- [3.4]{.pnum} For the overloads with no [`ExecutionPolicy`]{.rm}[execution policy]{.add}, at most
+- [3.4]{.pnum} [For the overloads with no `ExecutionPolicy`, a]{.rm}[A]{.add}t most
   `min(last1 - first1,  last2 - first2)` applications of the corresponding predicate and any projections.
-- [3.5]{.pnum} For the overloads with an [`ExecutionPolicy`]{.rm}[execution policy]{.add},
-  `@_O_@(min(last1 - first1, last2 - first2))` applications of the corresponding predicate.
+- [3.5]{.pnum} [For the overloads with an `ExecutionPolicy`,
+  `@_O_@(min(last1 - first1, last2 - first2))` applications of the corresponding predicate.]{.rm}
```

Comment:
This change makes the complexity requirements for parallel implementation as strict ("at most") as for serial one. It's not a mere wording change. Was it properly discussed?

Reply:
OK, it seems it can be covered by blanket wording in [algorithms.parallel.overloads] - but it's better to double-check.

Reply:
Cool I didn't know about that. So, having O notation in the algorithms is sort of redundant because it just duplicates existing blanket wording.